### ISSUE TITLE
UsePaginationQuery - setPage does not seem to work without existing cache

### ISSUE THREAD

**Issue by e-lobo** [psteinroe/supabase-cache-helpers#175]
I tried to trigger [setPage](https://github.com/psteinroe/supabase-cache-helpers/blob/7918b1bbeb0050ef2677c9447528b5b3924481c8/packages/postgrest-swr/src/query/use-pagination-query.ts#L96) for a table but it does not seem to work as intended. 
I tried calling `nextPage` & `previousPage` and these work well..  (cache populated)
now i try again to call `setPage` and it works ie retrieves from cache.

so basically `setPage` does not attempt to go out and fetch new data and only looks for stale data.

**Comment by psteinroe** [comment]
fixed in the latest release. Thanks for reporting!
